fili-core/src/main/java/com/yahoo/bard/webservice/sql/helper/CalciteHelper.java

```diff
@@ -0,0 +1,161 @@
+// Copyright 2017 Yahoo Inc.
+// Licensed under the terms of the Apache license. Please see LICENSE.md file distributed with this work for terms.
+package com.yahoo.bard.webservice.sql.helper;
+
+import org.apache.calcite.adapter.jdbc.JdbcSchema;
+import org.apache.calcite.plan.RelTraitDef;
+import org.apache.calcite.rel.rel2sql.RelToSqlConverter;
+import org.apache.calcite.schema.SchemaPlus;
+import org.apache.calcite.sql.SqlDialect;
+import org.apache.calcite.sql.parser.SqlParser;
+import org.apache.calcite.sql.pretty.SqlPrettyWriter;
+import org.apache.calcite.tools.Frameworks;
+import org.apache.calcite.tools.Programs;
+import org.apache.calcite.tools.RelBuilder;
+
+import java.sql.Connection;
+import java.sql.SQLException;
+import java.util.List;
+
+import javax.sql.DataSource;
+
+/**
+ * Small utility class to help with connection to databases, building, and writing sql.
+ */
+public class CalciteHelper {
+    public static final String DEFAULT_SCHEMA = "PUBLIC";
```

Comment:
I don't think this is true for most but not all databases. Another reason this should be specified in the Table logic and not here